### DISCUSSION TITLE
Use outside panel?

### DISCUSSION THREAD

**Discussion by sinnbeck** [saade/filament-fullcalendar#120] [Q&A]
It seems that the new version is made to only be used inside a panel? The beta allowed it to be used in a plain livewire component as well. Is there a way to skip the usage of the panel plugin ?

**Comment by sinnbeck** [comment]
Got it working by registering the panel plugin and tweaking my setup a ton :)

**Reply by jesusLM94** [reply]
Can you share details on how you accomplish this? i'm using all Filament packages without the panel builder and I've been trying to use filament-fullcalendar without luck.
